recipes/conversational_voice_agent/requirements.txt

```diff
@@ -14,4 +14,9 @@ librosa==0.10.1
 gradio==4.3.0
 
 # Bark model from the GitHub repository
-git+https://github.com/suno-ai/bark.git@773624d26db84278a55aacae9a16d7b25fbccab8
\ No newline at end of file
+git+https://github.com/suno-ai/bark.git@773624d26db84278a55aacae9a16d7b25fbccab8
+
+# SpeechT5
+datasets
+torch
+soundfile
```

Comment:
All dependencies must be pinned in this kind of project. We don't want to wake up and realize something doesn't work due to updates.

Reply:
@adrianboguszewski you mean we need to specify the version for each dependency or remove the versions?